content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -148,8 +148,23 @@ Allowed customization are:
 
 * To provide an alternative `imageRepository` to be used instead of
   `k8s.gcr.io`.
-* To provide a `unifiedControlPlaneImage` to be used instead of different images for control plane components.
-* To provide a specific `etcd.image` to be used instead of the image available at`k8s.gcr.io`.
+* To set `useHyperKubeImage` to `true` to use images from HyperKube.
+* (For `etcd` only) To provide a specific `imageRepository` and `imageTag` to be used instead of the image available at`k8s.gcr.io`.
+
+An example from [kubeadm API v1beta1](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1#hdr-Kubeadm_init_configuration_types):
```

Comment:
please remove this example. this link:
https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1

already has examples.